examples/05_advanced/plot_age_group_prediction_cross_val.py

```diff
@@ -0,0 +1,104 @@
+"""
+Functional connectivity predicts age group
+==========================================
+
+This example compares different kinds of functional connectivity between
+regions of interest : correlation, partial correlation, and tangent space
+embedding.
+
+The resulting connectivity coefficients can be used to
+discriminate children from adults. In general, the **tangent kind**
+**outperforms** the standard correlations: see `Dadi et al 2019
+<https://www.sciencedirect.com/science/article/pii/S1053811919301594>`_
+for a careful study.
+"""
+
+###############################################################################
+# Load brain development fMRI dataset and MSDL atlas
+# -------------------------------------------------------------------
+# We study only 60 subjects from the dataset, to save computation time.
+from nilearn import datasets
+
+rest_data = datasets.fetch_development_fmri(n_subjects=60)
```

Comment:
`rest_data` might be misleading, here, since they're watching a movie. Maybe just `func_data` ?